Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/Dotnet6.0/.template.config/template.json

```diff
@@ -8,8 +8,8 @@
     "generatorVersions": "[1.0.0.0-*)",
     "groupIdentity": "Microsoft.AzureFunctions.ProjectTemplates",
     "precedence": "100",
-    "identity": "Microsoft.AzureFunctions.ProjectTemplate.CSharp.Isolated.3.x",
-    "shortName": "func",
+    "identity": "Microsoft.AzureFunctions.ProjectTemplate.CSharp.Isolated.Dotnet6.4.x",
```

Comment:
Any other thoughts on the `identity` name?

Worried this reads as Dotnet.6.4, so maybe this is better:

`Microsoft.AzureFunctions.ProjectTemplate.CSharp.Dotnet6.Isolated.4.x`